src/EjectionManager.sol

```diff
@@ -144,17 +171,25 @@ contract EjectionManager is IEjectionManager, OwnableUpgradeable{
         emit EjectorUpdated(_ejector, _status);
     }
 
+    /// @dev internal function to set the operator stake cap percent
+    function _setOperatorStakeCapPercent(uint8 _quorumNumber, uint256 _operatorStakeCapPercent) internal {
+        operatorStakeCapPercent[_quorumNumber] = _operatorStakeCapPercent;
+        emit OperatorStakeCapPercentSet(_quorumNumber, _operatorStakeCapPercent);
+    }
+
     /**
      * @notice Returns the amount of stake that can be ejected for a quorum at the current block.timestamp
      * @param _quorumNumber The quorum number to view ejectable stake for
      */
-    function amountEjectableForQuorum(uint8 _quorumNumber) public view returns (uint256) {
+    function amountEjectableForQuorum(uint8 _quorumNumber) public view returns (uint256 ejectableStake, uint256 totalStake) {
```

Comment:
It feels like it would be easier if we:
prevented them from registering / updating their stake to an amount greater than the stake cap in the first place instead of ejecting an amount of stake from the operator(s) after the fact

Reply:
Discussed offline. Variable names were confusing